src/main/java/dan200/computercraft/shared/pocket/apis/PocketAPI.java

```diff
@@ -78,11 +78,12 @@ public void shutdown()
                     {
                         if( !(m_computer.getEntity() instanceof EntityPlayer) )
                         {
-                            throw new LuaException( "Cannot find player" );
+                            return new Object[] { false, "Cannot find player" };
                         }
 
                         EntityPlayer player = (EntityPlayer) m_computer.getEntity();
                         InventoryPlayer inventory = player.inventory;
+                        if( !m_computer.inInventory() ) return new Object[] { false, "Not in an inventory" };
```

Comment:
Why is this check necessary? Why does m_computer.getEntity() return the player when the player isn't holding the item anymore? That's what needs to be fixed.